src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Contains the task list with operations such as
+ * add or delete to alter the list
+ */
+public class TaskList {
+
+    ArrayList<Task> list;
+    int count;
```

Comment:
The `ArrayList` object has the .size() property so you don't need to store a separate `count` in the `TaskList` class. 

```java
list.size(); //Returns the number of elements in the list
```